Expensify/App
# Add scripts and instructions for static iOS builds

### Details
This is something I figured out today while testing push notifications – running this script will build and install the app as it exists in your current branch. Because it's a static, or "release" build, it is like any other app on your phone where it can be closed and reopened without any connection to the metro process.

For contrast:

1. `npm run ios`
1. After it builds, close the app.
1. The metro process will die.
1. Try to open the app again, and you'll get a red screen saying that it could not reach the metro server.

v.s:

1. `npm run ios-static`
1. After it builds and installs, close the app.
1. Try to open the app again, and it will reopen.

### Fixed Issues
Fixes n/a

### Tests
1. Run `npm i && npm run ios-static`
1. The app should build and install.
1. Close the app.
1. Reopen the app. It should reopen and work.

### QA Steps
None.

### Tested On
